### PR TITLE
duplicity: update to 1.0.1

### PR DESCRIPTION
**Pull request by dkwo**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)

### CODE REVIEW

**Comment by paper42** [comment]
> what's up with wrksrc? what is the new recommended way?

> wrksrc should not be defined

it's not required anymore

File: srcpkgs/duplicity/template [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
  tar par2cmdline"
 short_desc="Encrypted bandwidth-efficient backup using the rsync algorithm"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="dkwo <npiazza@disroot.org>"
 license="GPL-2.0-or-later"
 homepage="https://duplicity.gitlab.io"
-changelog="${homepage}/vers8/CHANGELOG.html"
-distfiles="https://launchpad.net/duplicity/${version%.*}-series/${version}/+download/duplicity-${version}.tar.gz"
-checksum=b24aec0df4849a71de64b743fe9957f09bb80259898e455bdc612c78bdb374e2
+changelog="https://duplicity.gitlab.io/vers8/CHANGELOG.html"
```
Comment (classabbyamp) [inline srcpkgs/duplicity/template:15]: 404

**Comment by dkwo** [comment]
also updated to newer version

File: srcpkgs/duplicity/template [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
  tar par2cmdline"
 short_desc="Encrypted bandwidth-efficient backup using the rsync algorithm"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="dkwo <npiazza@disroot.org>"
 license="GPL-2.0-or-later"
 homepage="https://duplicity.gitlab.io"
-changelog="${homepage}/vers8/CHANGELOG.html"
-distfiles="https://launchpad.net/duplicity/${version%.*}-series/${version}/+download/duplicity-${version}.tar.gz"
-checksum=b24aec0df4849a71de64b743fe9957f09bb80259898e455bdc612c78bdb374e2
+changelog="https://gitlab.com/duplicity/duplicity/-/blob/main/CHANGELOG.md"
```
Comment (classabbyamp) [inline srcpkgs/duplicity/template:15]:
```suggestion
changelog="https://gitlab.com/duplicity/duplicity/-/raw/main/CHANGELOG.md"
```

**Comment by paper42** [comment]
just for reference, this broke version reporting and deja-dup wasn't happy with that, @classabbyamp fixed this in https://github.com/void-linux/void-packages/commit/27a46b8d6d727bf252b8bcc00b43b4f420a26416

**Pull request merged**